### DISCUSSION TITLE
7B9D8513-6731-99A3-9624-86734FB9C7F9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#121] [General]
# 7B9D8513-6731-99A3-9624-86734FB9C7F9

Kepler-90 is a sun-like star located about 2,840 light-years from Earth in the constellation Draco. It is notable for possessing a planetary system that has the same number of observed planets as the Solar System

https://celestia.mobi/addon?item=7B9D8513-6731-99A3-9624-86734FB9C7F9

**Comment by exocritic5457** [comment]
A part of me feels like bci should have lava that close, especially with the several planets and presumed tidal activity. (Also there's a duplicate, planetless star evident in the star browser making it overall brighter in appearance, no Z-fighting)

I appreciate how much this resembles the impression though. Not too many addons do it, but when they do I find it cool as well

**Reply by MrSpace43** [reply]
Yeah I was thinking of adding lava to the inner planets, but left it out as the original impression doesn't show them with lava. I probably might give them lava when I redo the system to look a little more realistic, because this is one of my older addons, and I've improved quite a bit since I made this addon.

As for the duplicate star, that was because this addon was for desktop users for people who didn't have TTarrants' exoplanet catalog (default Celestia didn't have the Kepler-90 at the time when I made the addon).

Now that Celestia is on mobile devices, and that it does have Kepler-90, I'll fix that issue and remove the duplicate star when I get the chance.

**Reply by exocritic5457** [reply]
Such changes would go appreciated, and I'm patient enough since you can't rush art. Now with the addon, the planets orbit said star 48.9 AU away from the true, F9 one due to the aforementioned catalog shenanigans
